src/visualization/TransitSupply.vue

```diff
@@ -105,6 +105,7 @@ const store: any = {
   project: {},
```

Comment:
As with NetworkVolumePlot.vue i was not able to fully test the applied changes due to missing input files. @billyc please make sure this still works